v19.2/foreign-key.md

```diff
@@ -251,12 +251,11 @@ The update to the referenced table returns an error because `id = 1001` is refer
 > SELECT * FROM customers;
 ~~~
 ~~~
+   id  |         email
 +------+------------------------+
-|  id  |         email          |
-+------+------------------------+
-| 1001 | a@co.tld               |
-| 1111 | info@cockroachlabs.com |
-+------+------------------------+
+  1001 | a@co.tld
+  1111 | info@cockroachlabs.com
```

Comment:
~Same comment as above re: batching style changes.  It makes the reviews more costly in terms of time & attention.